src/app/modules/auth/login.page.tsx

```diff
@@ -27,6 +32,11 @@ function LoginPage() {
 
  const { login, loading } = useLogin({
   onCompleted: (data) => {
+   const user = data.login.user;
+
+   addUser({ name: user.name, id: user.id });
```

Comment:
Como as propriedades do objeto tem o mesmo nome (id & name) poderia simplificar assim:

```suggestion
   addUser(user);
```

Reply:
A tipagem esperada não é a mesma dai o typescript não deixa pois tem outros campos no retorno de user